owa/app/css/openmrs-owa-idgen.css

```diff
@@ -17,13 +17,16 @@ body {
     font-size: 16px;
 }
 
+html body {
```

Comment:
why was this added?

Reply:
obviously to center the body regardless of screen size

Reply:
have you checked the effects it has on the other tabs apart from yours

Reply:
<img width="1275" alt="screen shot 2017-10-29 at 10 11 48" src="https://user-images.githubusercontent.com/28532877/32141443-9f331a18-bc91-11e7-8c99-1a8ad0735d0a.png">

Reply:
this is how it looks in full screen. please remove it

Reply:
if you noticed, I removed the left: 140px property that you had put on most elements in the layout. This is because it rendered the page contents 140px away from the left creating the illusion that it is centred on all devices with screen sizes like ours but in actuality it wasnt

Reply:
Yes seen it  looks good

Reply:
also don't forget to remove the left: 140px in breadcrumbs.css

Reply:
I already removed it and committed the changes.

Reply:
All good @cnagadya